### DISCUSSION TITLE
Getting started with contributions!

### DISCUSSION THREAD

**Discussion by ShubhamTiwari55** [ruxailab/RUXAILAB#540] [General]
Hello everyone! I am Shubham Tiwari, a 3rd-year undergraduate in computer science and engineering from NIT Patna. I am willing to contribute to the organization. So how can I get started?

**Comment by marcgc21** [comment]
Hi @ShubhamTiwari55 , we are closing the creation of threads inside Discussions because it is hard to maintain for the organization. 

We prefer to work on Discord channels. There you will find the best place to know how to contribute to the organization, the repositories and also GSOC or other calls that we are running.
